cli/make-register.js

```diff
@@ -112,7 +132,9 @@ const lastActionRankMapping = {
 };
 
 // add fixture list sorted by lastModifyDate
-register.lastUpdated = Object.keys(register.filesystem).sort((a, b) => {
```

Comment:
Maybe ``!(`redirectTo` in register.filesystem[fixtureKey])`` would be clearer?

Reply:
But we actually use `lastModifyDate` and `lastAction` inside the following sort function, so filtering for those (assuming they only come in pairs) makes more sense IMO.